docs/user_guide/file_management.rst

```diff
@@ -0,0 +1,19 @@
+File Management
+===============
+
+User Quotas
+-----------
+Data importers that use the Tripal API and Tripal supported widgets automatically associate uploaded files with users. If you are allowing end-users to upload files you may want to consider adding quotas to prevent the server storage from filling.  To ensure that users do not exceed the limits of the server a quota system is available.  Navigate to **Administer > Tripal > File Management** and click the **User Quota** tab to reveal the following page:
+
+.. image:: ./file_management.user_quotas.1.png
```

Comment:
unfortunately this screenshot is for Galaxy.  Note the breadcrumb and URLs dont match.  would be better to replace with the core breadcrumbs if possible?

Reply:
very minor detail though since the actual quota system is identical

Reply:
Oops. I'll fix that. This code was originally in the Galaxy module and got moved to Tripal core.  I kept the screenshots because I thought they were clean of Galaxy stuff but apparently not.